Src/Sentry.Xamarin.Forms/Sentry.Xamarin.Forms.csproj

```diff
@@ -1,5 +1,5 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.0.41">
-	<Import Project="Directory.Build.props" />
+	<Import Project="../Directory.Build.props" />
```

Comment:
There's no need to add this, msbuild looks upwards already for it.

This is used when you have more than 1 of such files. Msbuild stops looking when it finds the first, so if you want one of the root, one on `src`, then the one on `src` must reference the root one explicitly (this is done on sentry-dotnet)